src/common/is-node-attached.js

```diff
@@ -1,11 +1,11 @@
 // Marionette.isNodeAttached
 // -------------------------
 
-import Backbone from 'backbone';
+import domMixin from '../mixins/dom';
 
 // Determine if `el` is a child of the document
 const isNodeAttached = function(el) {
-  return Backbone.$.contains(document.documentElement, el);
+  return domMixin.containsEl(document.documentElement, el);
```

Comment:
Assuming Node.contains is implemented (even ie5 has it according to MDN) 

Passing document.documentElement and el to $.contains,  following code path will be executed: 

`adown.contains( bup )`

where adown is document.documentElement and bup is el.parentNode

So the only difference would be, that using $.contains,  el.parentNode will be passed to Node.contains instead of el. Probably because of [this IE bug](https://connect.microsoft.com/IE/feedback/details/780874/node-contains-is-incorrect)

In fact using bellow code would execute the same code path as using jquery

`return document.documentElement.contains(el && el.parentNode)`